policies/late_night.md

```diff
@@ -0,0 +1,32 @@
+# Late Night Policy
+
+Last Updated: **NOT CURRENTLY RATIFIED  POLICY OF FARSET LABS**
+
+Next Review: **NOT CURRENTLY RATIFIED  POLICY OF FARSET LABS**
+
+## Late Events
+
+All events or gatherings which involve alcohol or loud music must end by 10.30 and vacate the building by 11pm. 
+
+Occasional members only gatherings may exceed this time but any loud music/PA must be disabled by 10.30
+
+## Late Guests
+
+**Non-member social guests are not permitted on the premises after 11pm.** If members are working collaboratively with non-members into the night that is acceptable. Members working collaboratively late at night with non-members must take responsibility for the non-members,
+including ensuring the number of non-members does not exceed the number of members working and alcohol consumption is kept to a minimum. This applies at all times but is particularly important at night.
+
+**Farset Labs should NEVER be an option for drinks after bars are closed.** Groups of drunk people arriving at our studio after chucking out time is not appropriate in a residential area and jeopardises the good will within the community for everyone. Absolutely NO exceptions.
+
+By all means have a few drinks if you're working late, but outside community run events, it's not OK to just hang out in the space areas drinking.
```

Comment:
Potentially refactor extend to clarify that it's **unacceptable**, and may result in disciplinary action